### ISSUE TITLE
Problem if subcollection contains deeper subcollection with same name

### ISSUE THREAD

**Issue by jimvine** [mickstar/Zoo-For-Zotero#56]
EDITED - I think I've worked out what the actual behaviour is here. It's not that items are not showing up, it's that the whole wrong subcollection is being displayed.

I have a subcollection that, nested within it has another (deeper level) subcollection of the same name. If I go into the highest subcollection it actually shows the items for the deeper one.

I.e., if I have a folder structure like:

```
Projects
    > Foo
        - Item 1
        - Item 2
        > Sub-foo
            >Bar
                - Item 3
            >Baz
                - Item 4
            >Foo
                - Item 5
                - Item 6
```
If I go into Collection `Projects` and then subcollection `Foo` I will see `Item 5` and `Item 6`, **not** items `Item 1` and `Item 2` and subcollection `Sub-foo` as expected.

(In my case, `Foo` is the name of a project, `Sub-foo` is actually `Definitions`, and then `Bar`, `Baz`, `Foo` etc. are terms related to 'Foo' that I want to gather definitions on. Hence why the term appears within itself, because I want a subcollection to store definitions of that term, and other subcollections to store definitions of related terms.)

**Comment by mickstar** [comment]
Yes I think you're correct. I believe the implementation is that zoo uses the collection name to lookup the collection, so if two collections have the same title it may get the wrong one.

There is no trivial solution as I'm aware, as my implementation for the lists of items is rather limited. I will look into it and try to have a solution for the next release.

**Comment by mickstar** [comment]
should be resolved here 6f9681bb793d9900afec4cddf2c95dcd033e5925
i pushed the update to the play store, should come within a week

**Comment by jimvine** [comment]
Thanks - following an updated version coming down from the Google Play store this seems to work correctly for me now!
